standalone/src/test/java/io/pravega/local/AuthEnabledInProcPravegaClusterTest.java

```diff
@@ -0,0 +1,155 @@
+    }
+
+    // Note: Strictly speaking, this test is really an "integration test" and is a little
+    // time consuming. For now, its intended to run as a unit test, but it could be moved
+    // to an integration test suite if and when necessary.
+    @Test(timeout = 50000)
+    public void testWriteAndReadEventWhenConfigIsProper() throws ReinitializationRequiredException {
+        String scope = "org.example.auth";
+        String streamName = "stream1";
+        int numSegments = 10;
+
+        ClientConfig clientConfig = ClientConfig.builder()
+                .credentials(new DefaultCredentials("1111_aaaa", "admin"))
+                .controllerURI(URI.create(localPravega.getInProcPravegaCluster().getControllerURI()))
+                .build();
+        log.info("Done creating client config");
+
+        @Cleanup
+        StreamManager streamManager = StreamManager.create(clientConfig);
+        log.info("Created a stream manager");
+
+        streamManager.createScope(scope);
+        log.info("Created a scope [{}]", scope);
+
+        streamManager.createStream(scope, streamName, StreamConfiguration.builder()
+                .scalingPolicy(ScalingPolicy.fixed(numSegments))
+                .build());
+        log.info("Created a stream with name [{}]", streamName);
+
+        @Cleanup
+        ClientFactory clientFactory = ClientFactory.withScope(scope, clientConfig);
+
+        @Cleanup
+        EventStreamWriter<String> writer = clientFactory.createEventWriter(streamName,
+                new JavaSerializer<String>(),
+                EventWriterConfig.builder().build());
+        log.info("Got a writer");
+
+        String writeEvent1 = "This is event 1";
+        writer.writeEvent(writeEvent1);
+        log.info("Done writing event [{}]", writeEvent1);
+
+        String writeEvent2 = "This is event 2";
+        writer.writeEvent(writeEvent2);
+        log.info("Done writing event [{}]", writeEvent2);
+
+        // Now, read the events from the stream.
+
+        String readerGroup = UUID.randomUUID().toString().replace("-", "");
+        ReaderGroupConfig readerGroupConfig = ReaderGroupConfig.builder()
+                .stream(Stream.of(scope, streamName))
+                .disableAutomaticCheckpoints()
+                .build();
+
+        @Cleanup
+        ReaderGroupManager readerGroupManager = ReaderGroupManager.withScope(scope, clientConfig);
+        readerGroupManager.createReaderGroup(readerGroup, readerGroupConfig);
+
+        @Cleanup
+        EventStreamReader<String> reader = clientFactory.createReader(
+                "readerId", readerGroup,
+                new JavaSerializer<String>(), ReaderConfig.builder().build());
+
+        // Keeping the read timeout large so that there is ample time for reading the event even in
+        // case of abnormal delays in test environments.
+        String readEvent1 = reader.readNextEvent(20000).getEvent();
+        log.info("Done reading event [{}]", readEvent1);
```

Comment:
This is a test, Why do you log?

Reply:
I do want to keep the log statements as the test brings up an in-memory cluster and is not totally unlikely to fail due to environmental factors. When a failure happens, the log statements might come in handy - especially when the build is tested after the merge in the Kubernetes cluster. 
What do you say?